### ISSUE TITLE
💅 `useConsistentCurlyBraces` doesn't report `prop={\n'string'\n}`

### ISSUE THREAD

**Issue by fregante** [biomejs/biome#7320]
### Environment information

```bash
CLI:
  Version:                      2.2.0
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v22.18.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         bun/1.2.18

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.jsonc
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               false
  VCS enabled:                  true

Linter:
  JavaScript enabled:           unset
  JSON enabled:                 unset
  CSS enabled:                  unset
  GraphQL enabled:              unset
  Recommended:                  false
  Enabled rules:
    correctness/noUnusedImports
    style/useConsistentCurlyBraces
    style/useExportType
    style/useImportType
    style/useNodejsImportProtocol

Workspace:
  Open Documents:               0
```

### Rule name

useConsistentCurlyBraces

### Playground link

https://biomejs.dev/playground/?lintRules=useConsistentCurlyBraces&analyzerFixMode=safeAndUnsafeFixes&code=CgA8AEMAbwBuAG4AZQBjAHQAZQBkAEYAaQBlAGwAZABUAGUAbQBwAGwAYQB0AGUACgAgACAAZABlAHMAYwByAGkAcAB0AGkAbwBuADEAPQAKACAAIAAgACAAJwBVAHAAbABvAGEAZAAgAGEAIABDAFMAVgAgAGYAaQBsAGUAIABjAG8AbgB0AGEAaQBuAGkAbgBnACAAYQBuACAAIgBlAG0AYQBpAGwAIgAgAGMAbwBsAHUAbQBuACwAIABhAG4AZAAgAG8AcAB0AGkAbwBuAGEAbAAgACIAZgBpAHIAcwB0AF8AbgBhAG0AZQAiACAAYQBuAGQAIAAiAGwAYQBzAHQAXwBuAGEAbQBlACIAIABjAG8AbAB1AG0AbgBzACcACgAgACAAZABlAHMAYwByAGkAcAB0AGkAbwBuADIAPQB7AAoAIAAgACAAIAAnAFUAcABsAG8AYQBkACAAYQAgAEMAUwBWACAAZgBpAGwAZQAgAGMAbwBuAHQAYQBpAG4AaQBuAGcAIABhAG4AIAAiAGUAbQBhAGkAbAAiACAAYwBvAGwAdQBtAG4ALAAgAGEAbgBkACAAbwBwAHQAaQBvAG4AYQBsACAAIgBmAGkAcgBzAHQAXwBuAGEAbQBlACIAIABhAG4AZAAgACIAbABhAHMAdABfAG4AYQBtAGUAIgAgAGMAbwBsAHUAbQBuAHMAJwAKACAAIAB9AAoAIAAgAGMAbABhAHMAcwBOAGEAbQBlAD0AewAKACAAIAAiAGQALQBiAGwAbwBjAGsAIgAKACAAIAB9AAoALwA%2BAA%3D%3D

### Expected result

`description2` should be reported just like `className` and then be autofixed to what's seen in `description1`

<img width="575" height="272" alt="Image" src="https://github.com/user-attachments/assets/b8d8edc7-cbfa-4c2b-83c6-c34b01dc5915" />

Semi-related issue:

- https://github.com/biomejs/biome/issues/7256

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
